include/openssl/ess.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018-2018 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+# include <openssl/ossl_typ.h>
```

Comment:
Please fix the prologue of this file :
```
 /*
  * Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
  *
  *
  * Licensed under the OpenSSL license (the "License").  You may not use
  * this file except in compliance with the License.  You can obtain a copy
  * in the file LICENSE in the source distribution or at
  * https://www.openssl.org/source/license.html
  */
 
 
 #ifndef HEADER_ESS_H
 # define HEADER_ESS_H
 
 # include <openssl/opensslconf.h>
 
# ifndef OPENSSL_NO_CMS

# ifdef  __cplusplus
extern "C" {
# endif
``` 
... 

don't forget the corresponding #endif 
These methods can be available without CMS support.